engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/TypeDefinitionSet.scala

```diff
@@ -0,0 +1,44 @@
+package pl.touk.nussknacker.engine
+
+import org.springframework.expression.spel.SpelNode
+import pl.touk.nussknacker.engine.definition.{DefinitionExtractor, ProcessDefinitionExtractor, TypeInfos}
+
+
+case class TypeDefinitionSet(typeDefinitions: Set[TypeInfos.ClazzDefinition] = Set()) {
+
+  def validateTypeReference(spelNode: SpelNode) : Boolean = {
+
+
+    if(spelNode.getChildCount < 1) {
+      //todo: przechwycić błąd w razie jakby nie miało jednego child node'a
+    }
+
+    val spelNodeChildAST = spelNode.getChild(0).toStringAST
+
+    typeDefinitions.foreach(typeDefinition => {
+      if(typeDefinition.clazzName.display.equals(spelNodeChildAST))
+        return true
+    })
+    false
+  }
+
+  def displayBasicInfo(spelNode : SpelNode) = {
```

Comment:
Are you sure this method should printing ? I think it should return string.

Reply:
changed it, thanks